symplyphysics/symbols/classical_mechanics.py

```diff
@@ -0,0 +1,96 @@
+"""
+Classical mechanics
+===================
+
+Symbols related to classical mechanics.
+"""
+
+from symplyphysics import units, SymbolNew, angle_type, clone_symbol, dimensionless
+
+force = SymbolNew("F", units.force)
+"""
+**Force** is a vector quantity denoting an influence that can cause an object to change its velocity
+unless counterbalanced by other forces.
+"""
+
+speed = SymbolNew("v", units.velocity)
+"""
+**Speed** is the rate of change of the object's position with respect to time. It is the scalar counterpart
+of the velocity vector, which is the rate of change of the object's displacement with respect to time.
+"""
+
+acceleration = SymbolNew("a", units.acceleration)
+"""
+**Acceleration** is the rate of change of the object's velocity with respect to time. It is a vector quantity.
+"""
+
+distance = SymbolNew("d", units.length)
+"""
+**Distance** is is a measure of the spatial separation between two points.
+"""
+
+radial_distance = SymbolNew("r", units.length)
+"""
+Distance to the origin of the coordinate system.
+"""
+
+distance_to_axis = SymbolNew("r", units.length)
+"""
+Distance to reference axis.
+"""
+
+length = SymbolNew("l", units.length)
```

Comment:
This looks quite basic to me. Maybe move it there?

Reply:
I suggest we don't overengineer now since we can re-order them later without changing anything in the documentation or modules.

Reply:
So in my opinion this whole separation of symbols into several files must be 'an implementation detail' and we shouldn't worry too much about what goes where.

Reply:
Ok. Eventually I want to move them to let users find them easily.

Reply:
Well I made it so that the user would, say, write `symbols.length` using the common namespace and not `symbols.classical_mechanics.length`.

Reply:
Common namespace for symbols is a separate issue.